modules/openapi-generator/src/main/resources/cpp-qt5-client/helpers-header.mustache

```diff
@@ -128,7 +149,7 @@ namespace {{this}} {
     bool fromJsonValue(QList<T> &val, const QJsonValue &jval) {
         bool ok = true;
         if(jval.isArray()){
-            for(const auto& jitem : jval.toArray()){
+            for(const auto jitem : jval.toArray()){
```

Comment:
Is there a reason for removing reference here?

Reply:
The compiler gives a warning that the method `QJsonValue .toArray()` cannot return a reference. So I removed it.

Reply:
Looks good to me then!